docs/topics.py

```diff
@@ -0,0 +1,68 @@
+"""
+This class will take in the path to the command_list.json file and the name of the command
+that the user wants a description/info of/on.
+
+the format of the output is as follows:
+[embed title, command description, permissions, parameter/(inputs) descriptions, command type]
+
+---
+
+the embed title is self explanatory: it is the title of the embed and in this case, the name of the command.
+
+the command description is the description of the command (duh).
+
+permissions is just a list of the permissions needed for the command to be executed
+
+parameter description is the description of the command's inputs. (i was personally thinking that
+the embed name should be something like "inputs" or "input descriptions")
+
+and the command type is just "Slash Command" and/or "User Context Menu"
+
+----------------------------------------------------------------
+
+quick code example:
+                                    the path      the command
+thelist = helpCommandOutput("./command_list.json", "verify").output()
+print(thelist)
+
+"""
+
+import orjson
+class helpCommandOutput:
+    def __init__(self, path, commandName):
```

Comment:
Minor nitpick, more of a reminder for myself to change later, but type hints and more standardized doc comment formats would make use in the main file simpler for new contributors.